recipes/tabula/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "tabula-py" %}
+{% set version = "1.1.1" %}
+{% set sha256 = "24a7febfa0d5debd3f5c2a8ad48f033b470fa3ef2176a6b0d2b691959a26d5ea" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - use-conda-tabula-java.patch
+
+build:
+  number: 0
+  script:
+    - rm tabula/*.jar  # [unix]
+    - del tabula\*.jar  # [win]
+    - python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - flake8
+  run:
+    - python
+    - pandas
+    - numpy
+    - requests
+    - distro  # [linux]
+    - tabula-java
+
+test:
+  imports:
+    - tabula
+
+about:
+  home: https://github.com/chezou/tabula-py
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
```

Comment:
@CurtLH, can you submit a PR upstream to have the license file added? In the meantime it is packaged under the recipe folder.

Reply:
I just created a [PR](https://github.com/chezou/tabula-py/pull/94) to include the license in the source tarball.